index.html

```diff
@@ -197,6 +197,51 @@ <h3 class="lang-h3">
         </ul>
       </div>
     </div>
+    <section class="contact-cont">
+      <div class="contact-margin">
+        <div class="contact-title">
+          <div class="contact-h1">
+            Get started
+          </div>
+          <div class="about-p p-height">
+            Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna.
+          </div>
+        </div>
+        <form class="contact-form" action="https://formspree.io/f/xnqrvgwb" method="post">
+            <ul class="contact-form-ul">
+              <li class="contact-form-li">
+                <label class="contact-form-li-label" for="user-name">Your name</label>
+                <input class="contact-form-li-input" type="text" name="user-name" id="user-name" maxlength="30" required placeholder="Enter your name here...">
+              </li>
+              <li class="contact-form-li">
+                <label class="contact-form-li-label" for="user-email">Your email</label>
+                <input class="contact-form-li-input" type="email" name="user-email" id="user-email" required placeholder="Enter your email here...">
+              </li>
+              <li class="contact-form-li contact-form-li-last-child">
+                <textarea name="message" class="contact-form-textarea" maxlength="500" required placeholder="Enter your message here..."></textarea>
+              </li>
+              <li>
+                <button class="start-btn" type="submit">Start collaboration</button>
```

Comment:
- [ OPTIONAL ] I suggest increasing the width for the button to match the Figma design as much as possible 🔥💯💪🏻

- Your design
![image](https://user-images.githubusercontent.com/58553711/193828552-8efa59f6-b2d8-428e-b7c3-be40ad58277f.png)

- Figma
![image](https://user-images.githubusercontent.com/58553711/193828644-4acc8202-dc48-45cf-b281-c6895821963b.png)